src/crimson/os/seastore/random_block_manager/nvme_block_device.cc

```diff
@@ -200,6 +200,22 @@ open_ertr::future<> NVMeBlockDevice::open_for_io(
   });
 }
 
+NVMeBlockDevice::mount_ret NVMeBlockDevice::mount()
+{
+  logger().debug(" mount ");
+  return open(device_path, seastar::open_flags::rw | seastar::open_flags::dsync
+  ).safe_then([this] {
+    return read_rbm_header(RBM_START_ADDRESS
```

Comment:
And why `EphemeralRBMDevice::mount(`) doesn't call `read_rbm_header()` to load the super? This implementation should be expected to be consistent between tests and normal-path, right?